src/24-uncensorable-wikipedia/index.md

```diff
@@ -0,0 +1,196 @@
+
+Your main options for accessing the snapshot of Wikipedia are: _(depending on your network, some of these won't work)_
+
+* **Option**: Use the ipfs.io gateway to access it: http://ipfs.io/ipns/tk-wikipedia-ipns/out/A/Anasayfa.html
+* **Option**: Connect with the IPFS network over Tor (this is experimental. Read [this tutorial on Tor gateways](https://flyingzumwalt.gitbooks.io/decentralized-web-primer/content/avenues-for-access/tor-gateways.html) and [this tutorial on using IPFS with Tor transport](https://flyingzumwalt.gitbooks.io/decentralized-web-primer/content/avenues-for-access/tor-transport.html))
+* **Option**: Install an IPFS node on your computer and access Wikipedia through that node _(requires [using the command line](http://lifehacker.com/5633909/who-needs-a-mouse-learn-to-use-the-command-line-for-almost-anything), but this is the most reliable method because it retrieves the content directly from the IPFS peer-to-peer network)_
+    1. Install IPFS [following these instructions]()
+    2. Start your IPFS node by running `ipfs daemon` so it can connect to the network.
+    3. Read the content through your IPFS node's HTTP gateway by visiting http://localhost:8080/ipns/tk-wikipedia-ipns/out/A/Anasayfa.html
+
+We've almost finished creating a web browser extension that will allow you to access IPFS directly from your web browser without installing any additional external tools. Watch https://github.com/ipfs/in-web-browsers for the announcement when we release that browser extension.
+
+## How to Mirror Wikipedia on an IPFS Node
+
+If you want to help make this information available, you can install IPFS on your computer and _"pin"_ the Wikipedia snapshot(s) on that IPFS node. Here are instructions for doing that.
+
+**Prerequisites**:
+* Requires [using the command line](http://lifehacker.com/5633909/who-needs-a-mouse-learn-to-use-the-command-line-for-almost-anything)
+* You need to have enough storage space to hold the snapshot(s) you want to serve.
+    * Turkish Wikipedia is about 15 GB
+    * English Wikipedia is about 150 GB
+
+If you don't have enough storage space to hold full copies of the snapshot(s), you can still run an IPFS gateway so that people can rely on you to retrieve the content from the IPFS network on the fly.
+
+**Steps**:
+   1. Install IPFS [following these instructions]()
+   2. Start your IPFS node by running `ipfs daemon` so it can connect to the network.
+   3. Pin the snapshot(s) onto your machine
+      - English Wikipedia `ipfs pin add en-wikipedia-hash`
+      - Turkish Wikipedia `ipfs pin add tk-wikipedia-hash`
+   4. TODO If possible, give your machine a public IP addresses
```

Comment:
Yeah, that could work. Though if your home internets nat traversal is working, it should work just fine.

I have a tool that checks nat traversal here: https://github.com/whyrusleeping/natest
But i need to write better docs for it